designsafe/apps/api/tasks.py

```diff
@@ -915,21 +915,23 @@ def set_facl_project(self, project_uuid, usernames):
         logger.debug('set facl project: {}'.format(res))
 
 @shared_task(bind=True, max_retries=3, default_retry_delay=60)
-def email_collaborator_added_to_project(self, project_title, project_url, team_members_to_add, co_pis_to_add):
-
+def email_collaborator_added_to_project(self, project_uuid, project_title, project_url, team_members_to_add, co_pis_to_add):
+    from designsafe.apps.projects.models.agave.experimental import ExperimentalProject
```

Comment:
As a heads-up, we have a lot of different project types (experimental, simulation, field research, and growing). You won't need to worry about it here if you use Celery chaining, but if you need to in the future you'll want to do a lookup to make sure you're using the correct model. There's an example of how to do it in the POST method of `designsafe/apps/api/projects/views.py` where `prj` is defined.